06-files-input-and-output/remove_files.py

```diff
@@ -2,33 +2,43 @@
 
 import os
 
-def find_files_in_folder(path, extention, file_max_size):
+def find_files_in_folder(path, extension, file_max_size):
+    print(
+        "Searching for files with .{} extension in \"{}\" and maximum size of {}bytes".format(
```

Comment:
When we have multiple placeholders for formatting I prefer to add names for the placeholders, and call the format function with keyword arguments.